src/Sylius/Component/Promotion/Message/CatalogPromotionCreated.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Promotion\Message;
```

Comment:
```suggestion
namespace Sylius\Component\Promotion\Event;
```
IMHO it should be in namespace `Event`, it will be more precise what it is.